src/components/SamlProviderConfiguration/index.jsx

```diff
@@ -267,15 +285,20 @@ class SamlProviderConfiguration extends React.Component {
   }
 }
 
-SamlProviderConfiguration.defaultProps = {
+SamlProviderConfigurationCore.defaultProps = {
   enterpriseSlug: null,
   enterpriseName: null,
 };
 
-SamlProviderConfiguration.propTypes = {
+SamlProviderConfigurationCore.propTypes = {
   enterpriseName: PropTypes.string,
   enterpriseSlug: PropTypes.string,
   enterpriseId: PropTypes.string.isRequired,
+  learnerPortalEnabled: PropTypes.bool.isRequired,
 };
 
-export default SamlProviderConfiguration;
+const mapStateToProps = state => ({
```

Comment:
could I possibly get a TLDR on what this is doing?

Reply:
ah sure, tldr: this injects redux state values into the component in the form of props. This is the react-redux way of fetching redux state into components. I know there is new react-redux versions that support hooks instead of this wrapped component approach but it was very tricky to upgrade last time I tried haha